### ISSUE TITLE
Postgres: Different UNIQUE notations

### ISSUE THREAD

**Issue by mpf82** [tobymao/sqlglot#387]
Postgres supports various ways to declare a column as unique.

sqlglot currently supports: 
```sql
CREATE TABLE products (
    product_no integer UNIQUE,
    name text,
    price numeric
);
```

It would be nice to also support:
```sql
CREATE TABLE example (
    a integer,
    b integer,
    c integer,
    UNIQUE (a, c)
);
```

and with a named constraint:
```sql
CREATE TABLE products (
    product_no integer CONSTRAINT must_be_different UNIQUE,
    name text,
    price numeric
);
```

Reference: https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-UNIQUE-CONSTRAINTS

**Comment by tobymao** [comment]
low priority, happy to help you accomplish this in a PR, or may take a week

**Comment by tobymao** [comment]
fixed by #405
